andela/theoden-cfh
# Chore/151136242/convert grunt to gulp

What does this PR do?

Converts from Grunt to Gulp Task runner, and fixes travis crash

Description of Task to be completed?

add grunt to package dependencies
create gulpfile.js with tasks equivalent to those in existing gruntfile
modify npm scripts for start, test and postinstall
delete gruntfile and remove grunt from package dependencies
How should this be manually tested?

Run npm start. It should run gulp instead of grunt.

Any background context you want to provide?

Grunt is an older task runner than Gulp; with Gulp, the syntax is much simpler to comprehend and it uses pipe streams which makes it inherently faster.

What are the relevant pivotal tracker stories?

#151136242

Screenshots (if appropriate)

Questions:
